chrome/content/newtab.css

```diff
@@ -122,7 +122,7 @@ url-prefix("about:home") {
     border-radius: 12px !important;
     height: 44px !important;
     width: 44px !important;
-    box-shadow: none !important;
+    box-shadow: rgb(0 0 0 / 0.1) !important;
```

Comment:
It seems when changing the colour you have removed the shadow's properties (offset and blur).

Reply:
This is my mistake, I have corrected it.